assets/scss/cards.scss

```diff
@@ -153,6 +153,7 @@
         overflow: hidden;
         display: -webkit-box;
         -webkit-line-clamp: 3;
+        text-transform: capitalize;
```

Comment:
I don't think we should do this. It will affect all contexts, not just the new entity listing page, and may well be gramatically incorrect in some languages.